content/en/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -32,8 +32,6 @@ Before choosing a guide, here are some considerations:
 
 Note: Not all distros are actively maintained. Choose distros which have been tested with a recent version of Kubernetes.
 
-If you are using a guide involving Salt, see [Configuring Kubernetes with Salt](/docs/admin/salt/).
```

Comment:
The link can be replaced with /docs/setup/salt

Reply:
ok